internal/pkg/cmd/prerunner.go

```diff
@@ -888,6 +888,12 @@ func warnIfConfluentLocal(cmd *cobra.Command) {
 	if strings.HasPrefix(cmd.CommandPath(), "confluent local kafka start") {
 		output.ErrPrintln("The local commands are intended for a single-node development environment only, NOT for production usage. See more: https://docs.confluent.io/current/cli/index.html")
 		output.ErrPrintln()
+		return
+	}
+	if strings.HasPrefix(cmd.CommandPath(), "confluent local") && !strings.HasPrefix(cmd.CommandPath(), "confluent local kafka") {
+		output.ErrPrintln("The local commands are intended for a single-node development environment only, NOT for production usage. See more: https://docs.confluent.io/current/cli/index.html")
+		output.ErrPrintln("As of Confluent Platform 8.0, Java 8 is no longer supported.")
```

Comment:
Wait, was this the old message?

CP 8.0 isn't out yet I think so this might be confusing.

Reply:
I think we wanted to warn people in advance?

Reply:
https://github.com/confluentinc/cli/pull/1513

Reply:
That makes sense, but then I would suggest something like `Java 8 will no longer be supported in Confluent Platform 8.0.` to suggest that it's something that will happen, and not something that has already happened.

Reply:
```suggestion
		output.ErrPrintln("As of Confluent Platform 8.0, Java 8 will no longer be supported.")
```